### PR TITLE
cli with --standalone option should work without --internal and --region

### PR DESCRIPTION
**Pull request by ailisp**
Issue: 
In appliance_console_cli, when given options `--standalone`, it should configure database without `--internal` and without `--region`. For example this should be correct:
```bash
appliance_console_cli --username root --password smartvm --dbname vmdb_production --standalone --key --dbdisk /dev/vdb
```
BZ:
https://bugzilla.redhat.com/show_bug.cgi?id=1513584

\cc @yrudman @carbonin  @gtanzillo 
@miq-bot add-label wip, bug

### CODE REVIEW

**Comment by ailisp** [comment]
@miq-bot remove-label wip

File: lib/manageiq/appliance_console/cli.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def host
 
     # database hostname
     def hostname
-      options[:internal] ? "localhost" : options[:hostname]
+      options[:standalone] || options[:internal] ? "localhost" : options[:hostname]
```
Comment (carbonin) [inline lib/manageiq/appliance_console/cli.rb]: I would rather see this change made in the definition of `#database?`

Instead of making `#hostname` be truthy when `--standalone` is specified, I think it makes more sense to make `#database` truthy when we are given a hostname or when `--standalone` is specified.

Comment (ailisp) [reply lib/manageiq/appliance_console/cli.rb]: Sounds good!

**Comment by ailisp** [comment]
@carbonin refactoring and still work

File: lib/manageiq/appliance_console/cli.rb [hunk 2]
```diff
@@ -262,7 +266,7 @@ def create_key
     def install_certs
       say "creating ssl certificates"
       config = CertificateAuthority.new(
-        :hostname => host,
+        :hostname => options[:standalone] ? "localhost" : host,
```
Comment (carbonin) [inline lib/manageiq/appliance_console/cli.rb]: Why was this changed?

I think `host` is the appliance hostname, not the database host.

**Comment by ailisp** [comment]
@carbonin sorry about that.

**Comment by ailisp** [comment]
@carbonin Finished specs now. Thanks

File: spec/cli_spec.rb [hunk 3]
```diff
@@ -431,6 +447,48 @@
       end
     end
 
+    context "#local_database?" do
+      it "should return false when no database given" do
+        expect(subject).to receive(:database?).and_return(false)
```
Comment (carbonin) [inline spec/cli_spec.rb]: I would prefer all of these stubs be done by just setting the options like what you're doing on [line 464](https://github.com/ManageIQ/manageiq-appliance_console/pull/20/files#diff-9f2c191ddaf44048040f9babca4ec4caR464)

Comment (ailisp) [reply spec/cli_spec.rb]: Thanks, but I'm not agree with this, since we unit test `local_database?`, I think it's better to assume `database?` and other stubs works correctly. For whether `database?` work correctly or not is should be tested in their own context. Is this understanding make sense?

Comment (carbonin) [reply spec/cli_spec.rb]: Makes sense to me.
